app/views/steps/applicant/personal_details/edit.html.erb

```diff
@@ -1,27 +1,30 @@
 <% title t('.page_title') %>
+<% step_header %>
 
-<div class="grid-row">
-  <div class="column-two-thirds">
-    <%= step_header %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= govuk_error_summary %>
 
-    <h1 class="heading-xlarge gv-u-heading-xxlarge"><%=t '.heading', name: @form_object.record.full_name %></h1>
+    <h1 class="govuk-heading-xl"><%=t '.heading', name: @form_object.record.full_name %></h1>
 
-    <%= step_form @form_object do |f| %>
-      <%=
-        f.radio_button_fieldset :has_previous_name, inline: true do |fieldset|
-          fieldset.radio_input GenericYesNo::YES, panel_id: :previous_name_panel
-          fieldset.radio_input GenericYesNo::NO
-          fieldset.revealing_panel(:previous_name_panel) { |panel| panel.text_field :previous_name }
-        end
-      %>
+    <%= step_form @form_object, builder: GOVUKDesignSystemFormBuilder::FormBuilder do |f| %>
+      <%= f.govuk_radio_buttons_fieldset :has_previous_name, legend: { tag: 'span', size: 's'  } do %>
+        <%= f.govuk_radio_button :has_previous_name, GenericYesNo::YES, link_errors: true do %>
+          <%= f.govuk_text_field :previous_name, label: { size: 's' } %>
+        <% end %>
+        <%= f.govuk_radio_button :has_previous_name, GenericYesNo::NO %>
+      <% end %>
 
-      <%= f.radio_button_fieldset :gender, choices: Gender.values %>
-      <%= f.gov_uk_date_field :dob, placeholders: true, legend_text: t('shared.form_elements.dob'), form_hint_text: t('shared.form_elements.dob_applicant_hint') %>
-      <%= f.text_field :birthplace %>
+      <%= f.govuk_radio_buttons_fieldset :gender, legend: { tag: 'span', size: 's' } do %>
```

Comment:
Like above, collections without anything revealing, better this way:
https://github.com/ministryofjustice/c100-application/blob/develop/app/views/steps/children/personal_details/edit.html.erb#L19